README.md

```diff
@@ -129,19 +130,32 @@ sign-offs to merge PRs (and for large or contentious issues we may
 wait for more). For typos or other small fixes to documentation we
 relax this to a single sign-off.
 
-### Related Projects
+### The cats ecosystem
 
-There are many projects that integrate with Cats:
+#### Generic libraries to support pure functional programming
 
- * [Circe](https://github.com/circe/circe): pure functional JSON library.
+ * [cats-effects](https://github.com/typelevel/cats-effect): provide a standard `IO` type for the cats ecosystem
+ * [cats-mtl](https://github.com/typelevel/cats-mtl): ??????
```

Comment:
In fact alleycats and mouse probably fall into that category as well.  But I am not sure how to name this category though. These projects all have different set of maintainers and thus are more independent from each other than just submodules of cats with different release cycle.